.github/workflows/m2k-func.yaml

```diff
@@ -99,7 +99,7 @@ jobs:
             ${PR_OR_COMMIT_URL} \
             "yq --inplace '.kfunction.image=\"quay.io/orchestrator/serverless-workflow-m2k-kfunc:${GITHUB_SHA}\"' charts/move2kube/values.yaml" \
             scripts/create_automated_m2k_kfunc_pr.sh \
-            ${{ github.event.head_commit.author.email || 'parodos@redhat.com' }} \
+            ${{ github.event.head_commit.author.email || 'rhdhorchestrator@redhat.com' }} \
```

Comment:
@pkliczewski do we have that email address set?

Reply:
not yet